\documentclass{bioinfo}
\copyrightyear{}
\pubyear{}
%\usepackage{scicite}
%\usepackage[blocks]{authblk}
%\usepackage[utf8x]{inputenc}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage[font=scriptsize,labelfont={bf}]{caption}
\usepackage[justification=raggedright,nearskip=10pt,farskip=0pt]{subfig}%[FIGTOPCAP, raggedright, hang, nooneline]{subfigure}
%\usepackage{subfig}
\usepackage{calc}
\usepackage{url}
\usepackage{array}
\usepackage{rotating}
\usepackage{tabularx}
%\usepackage{wasysym}
\usepackage{multirow}
%\usepackage[title]{appendix}
%\usepackage[authoryear]{natbib}
%\usepackage{cite}
%\usepackage{named}
%\usepackage{setspace}
\usepackage{layouts}
%\usepackage{titlesec}
%\usepackage[multiple]{footmisc}
%\usepackage{enumitem}
\usepackage{titletoc}
\usepackage{setspace}
\usepackage{longtable}

\hbadness=9999999 %avoid complaining for underfull h/vbox
\vbadness=9999999

% \setlength{\columnwidth}{86mm}
 \captionsetup[subfloat]{position=top, justification=raggedright, singlelinecheck=false, labelformat=simple, font={scriptsize,sf,bf}}
 \renewcommand*\thesubfigure{\Alph{subfigure}}
 \setlength{\belowcaptionskip}{-3mm}
 %\setlength{\abovecaptionskip}{0mm}
%\addtolength{\abovecaptionskip}{-3mm}

%\subfiguretopcaptrue

\pdfminorversion=4
\pdfobjcompresslevel=0

\usepackage{pgf}
\usepackage{tikz}
\usetikzlibrary{shapes,arrows}

\usepackage{footnote}

\pgfdeclareimage[height=3ex]{win-logo}{images/win_logo}
\pgfdeclareimage[height=3ex]{mac-logo}{images/mac_logo}
\pgfdeclareimage[height=3ex]{linux-logo}{images/linux_logo}
\pgfdeclareimage[height=3ex]{ubuntu-logo}{images/ubuntu_logo}
\pgfdeclareimage[height=2ex]{mac-command}{images/mac_command}
\pgfdeclareimage[height=2ex]{lower-bound}{images/lowerbound}
\pgfdeclareimage[height=2ex]{upper-bound}{images/upperbound_green}

\newcommand{\winsymbol}{\raisebox{-0.8ex}{\pgfuseimage{win-logo}}}
\newcommand{\macsymbol}{\raisebox{-0.5ex}{\pgfuseimage{mac-logo}}}
\newcommand{\linuxsymbol}{\raisebox{-0.8ex}{\pgfuseimage{linux-logo}}}
\newcommand{\ubuntusymbol}{\raisebox{-0.9ex}{\pgfuseimage{ubuntu-logo}}}
\newcommand{\winmaclinux}[3]{\begin{itemize}
    \item[\winsymbol] #1
    \item[\macsymbol] #2
    \item[\linuxsymbol] #3
\end{itemize}}
\newcommand{\macpc}[2]{\begin{itemize}
    \item[\macsymbol] #1
    \item[\winsymbol\ \  \linuxsymbol] #2
\end{itemize}}

\newcommand{\maccmd}{\raisebox{-0.35ex}{\pgfuseimage{mac-command}}}

\setlength{\abovecaptionskip}{1.3mm}
%\setlength{\belowcaptionskip}{1mm}

\def\ta{Timed Automaton}
\def\tas{Timed Automata}
  

\makeindex

\begin{document}
\addtocontents{toc}{\protect\setcounter{tocdepth}{-1}}
\firstpage{1}


\title[Bringing biological networks to life with ANIMO]{Bringing biological networks to life with ANIMO}

\author[Schivo, Scholma \textit{et~al}]{Stefano Schivo\,$^{1,*}$, Jetse Scholma\,$^{2,*}$, Brend Wanders\,$^3$, Ricardo A. Urquidi Camacho\,$^2$, Paul E. van der Vet\,$^3$, Marcel Karperien\,$^2$, Rom Langerak\,$^1$, Jaco van de Pol\,$^{1,\ddag}$ and Janine N. Post\,$^{2,\ddag}$}

\address{$^{1}$Formal Methods and Tools, Faculty of EEMCS, University of Twente, 7522NH Enschede, The~Netherlands;
$^{2}$Developmental BioEngineering, MIRA Institute for Biomedical Technology and Technical Medicine, University of Twente, 7522NH Enschede, The~Netherlands;
$^{3}$Human Media Interaction, Faculty of EEMCS, University of Twente, 7522NH Enschede, The~Netherlands}


\history{}%Received on XXXXX; revised on XXXXX; accepted on XXXXX}

\editor{}%Associate Editor: XXXXXXX}

\maketitle


\begin{abstract}
\input{abstract.tex}
\end{abstract}

\let\oldthefootnote\thefootnote
\renewcommand{\thefootnote}{\fnsymbol{footnote}}
\footnotetext[1]{These authors contributed equally to this work.}
\footnotetext[3]{To whom correspondence should be addressed.}
\let\thefootnote\oldthefootnote


\section{Introduction}\label{sec:introduction}
\input{introduction.tex}


\begin{methods}
\section{Methods}
\input{methods.tex}
\end{methods}




\section{Results}\label{sec:results}
\input{results.tex}




\section{Discussion}
\input{discussion.tex}

\subsection*{Acknowledgements}
We would like to thank Christof Franke for the useful discussions.
\paragraph*{Funding\textcolon} we acknowledge support from Netherlands Organisation for Scientific Research (NWO) Casimir grant 018.003.031 to J. S.
\paragraph*{Conflict of interest\textcolon} none declared.



% \bibliographystyle{apalike}
% {\scriptsize
% \bibliography{Paper}
% }
\begin{thebibliography}{}

\bibitem[Aldridge et~al., 2009]{pathway-fuzzy}
Aldridge, B.~B., Saez-Rodriguez, J., Muhlich, J.~L., Sorger, P.~K., and
  Lauffenburger, D.~A. (2009).
\newblock {Fuzzy logic analysis of kinase pathway crosstalk in
  TNF/EGF/Insulin-induced signaling}.
\newblock {\em PLoS Comput Biol}, 5(4):e1000340+.

\bibitem[Alur and Dill, 1994]{timed-automata-alur-dill}
Alur, R. and Dill, D.~L. (1994).
\newblock A theory of timed automata.
\newblock {\em Theor. Comput. Sci.}, 126(2):183--235.

\bibitem[Bailly et~al., 1996]{ap1-il1a}
Bailly, S., Fay, M., Isra{\"{e}}l, N., and Gougerot-Pocidalo, M.~A. (1996).
\newblock The transcription factor {AP-1} binds to the human interleukin 1
  alpha promoter.
\newblock {\em European Cytokine Network}, 7(2):125--128.

\bibitem[Bannister et~al., 1994]{cfos-cjun}
Bannister, A.~J., Brown, H.~J., Sutherland, J.~A., and Kouzarides, T. (1994).
\newblock {Phosphorylation of the c-Fos and c-Jun HOB1 motif stimulates its
  activation capacity.}
\newblock {\em Nucleic Acids Res}, 22(24):5173--6+.

\bibitem[Bartocci et~al., 2009]{bartocci-oscillators}
Bartocci, E., Corradini, F., Merelli, E., and Tesei, L. (2009).
\newblock Model checking biological oscillators.
\newblock {\em Electronic Notes in Theoretical Computer Science},
  229(1):41--58.
\newblock Proceedings of the Second Workshop From Biology to Concurrency and
  Back (FBTC 2008).

\bibitem[Batt et~al., 2007]{oded-ode-ta-discretization}
Batt, G., Salah, R.~B., and Maler, O. (2007).
\newblock On timed models of gene networks.
\newblock In {\em Proceedings of the 5th international conference on Formal
  modeling and analysis of timed systems}, FORMATS'07, pages 38--52, Berlin,
  Heidelberg. Springer-Verlag.

\bibitem[Bengtsson et~al., 2002]{ta-audio-protocol}
Bengtsson, J., Griffioen, W.~D., Kristoffersen, K.~J., Larsen, K.~G., Larsson,
  F., Pettersson, P., and Yi, W. (2002).
\newblock Automated verification of an audio-control protocol using {UPPAAL}.
\newblock {\em The Journal of Logic and Algebraic Programming}, 52-53(0):163 --
  181.

\bibitem[Bonzanni et~al., 2009]{petri-nets2}
Bonzanni, N., Krepska, E., Feenstra, K.~A., Fokkink, W., Kielmann, T., Bal,
  H.~E., and Heringa, J. (2009).
\newblock Executing multicellular differentiation: quantitative predictive
  modelling of c.elegans vulval development.
\newblock {\em Bioinformatics/computer Applications in The Biosciences},
  25:2049--2056.

\bibitem[Ciocchetta and Hillston, 2009]{bio-pepa}
Ciocchetta, F. and Hillston, J. (2009).
\newblock {Bio-PEPA: A framework for the modelling and analysis of biological
  systems}.
\newblock {\em Theor. Comput. Sci.}, 410:3065--3084.

\bibitem[Davis, 2000]{jnk-signaling}
Davis, R.~J. (2000).
\newblock Signal transduction by the {JNK} group of {MAP} kinases.
\newblock {\em Cell}, 103(2):239--252.

\bibitem[{de Jong}, 2002]{hidde-review}
{de Jong}, H. (2002).
\newblock Modeling and simulation of genetic regulatory systems: A literature
  review.
\newblock {\em Journal of Computational Biology}, 9:67--103.

\bibitem[Dematt\'{e} et~al., 2008]{blenx}
Dematt\'{e}, L., Priami, C., and Romanel, A. (2008).
\newblock {Modelling and simulation of biological processes in BlenX}.
\newblock {\em SIGMETRICS Perform. Eval. Rev.}, 35:32--39.

\bibitem[D{\'{e}}rijard et~al., 1994]{ras-jnk1}
D{\'{e}}rijard, B., Hibi, M., Wu, I.-H., Barrett, T., Su, B., Deng, T., Karin,
  M., and Davis, R.~J. (1994).
\newblock {JNK1}: A protein kinase stimulated by uv light and ha-ras that binds
  and phosphorylates the c-{Jun} activation domain.
\newblock {\em Cell}, 76(6):1025--1037.

\bibitem[Efroni et~al., 2003]{interacting-sm2}
Efroni, S., Harel, D., and Cohen, I.~R. (2003).
\newblock Toward rigorous comprehension of biological complexity: Modeling,
  execution, and visualization of thymic t-cell maturation.
\newblock {\em Genome Research}, 13(11):2485--2497.

\bibitem[Fathallah-Shaykh et~al., 2009]{drosophila-ode-model}
Fathallah-Shaykh, H.~M., Bona, J.~L., and Kadener, S. (2009).
\newblock Mathematical model of the drosophila circadian clock: Loop regulation
  and transcriptional integration.
\newblock {\em Biophysical Journal}, 97(9):2399 -- 2408.

\bibitem[Fisher et~al., 2005]{interacting-sm1}
Fisher, J., Piterman, N., Hubbard, E. J.~A., Stern, M.~J., and Harel, D.
  (2005).
\newblock Computational insights into caenorhabditis elegans vulval
  development.
\newblock {\em Proceedings of the National Academy of Sciences of the United
  States of America}, 102(6):1951--1956.

\bibitem[Gaudet et~al., 2005]{pathway-compendium}
Gaudet, S., Janes, K.~A., Albeck, J.~G., Pace, E.~A., Lauffenburger, D.~A., and
  Sorger, P.~K. (2005).
\newblock {A compendium of signals and responses triggered by prodeath and
  prosurvival cytokines.}
\newblock {\em Mol Cell Proteomics}, 4(10):1569--1590.

\bibitem[Hessel and Pettersson, 2007]{ta-wap-gateway}
Hessel, A. and Pettersson, P. (2007).
\newblock Model-based testing of a wap gateway: an industrial case-study.
\newblock In {\em Proceedings of the 11th international workshop, FMICS 2006
  and 5th international workshop, PDMC conference on Formal methods:
  Applications and technology}, FMICS'06/PDMC'06, pages 116--131, Berlin,
  Heidelberg. Springer-Verlag.

\bibitem[Hornbeck et~al., 2004]{phosphosite}
Hornbeck, P.~V., Chabra, I., Kornhauser, J.~M., Skrzypek, E., and Zhang, B.
  (2004).
\newblock {PhosphoSite}: A bioinformatics resource dedicated to physiological
  protein phosphorylation.
\newblock {\em Proteomics}, 4(6):1551--1561.

\bibitem[Janes et~al., 2005]{pathway-leastsquare}
Janes, K.~A., Albeck, J.~G., Gaudet, S., Sorger, P.~K., Lauffenburger, D.~A.,
  and Yaffe, M.~B. (2005).
\newblock A systems model of signaling identifies a molecular basis set for
  cytokine-induced apoptosis.
\newblock {\em Science}, 310(5754):1646--1653.

\bibitem[Janes et~al., 2006]{pathway-autocrine}
Janes, K.~A., Gaudet, S., Albeck, J.~G., Nielsen, U.~B., Lauffenburger, D.~A.,
  and Sorger, P.~K. (2006).
\newblock The response of human epithelial cells to {TNF} involves an inducible
  autocrine cascade.
\newblock {\em Cell}, 124(6):1225--1239.

\bibitem[Kanehisa and Goto, 2000]{kegg}
Kanehisa, M. and Goto, S. (2000).
\newblock {KEGG: Kyoto Encyclopedia of Genes and Genomes}.
\newblock {\em Nucleic Acids Research}, 28(1):27--30.

\bibitem[Killcoyne et~al., 2009]{cytoscape}
Killcoyne, S., Carter, G.~W., Smith, J., and Boyle, J. (2009).
\newblock {Cytoscape: a community-based framework for network modeling.}
\newblock {\em Methods in molecular biology (Clifton, N.J.)}, 563:219--239.

\bibitem[Larsen et~al., 1997]{uppaal}
Larsen, K.~G., Pettersson, P., and Yi, W. (1997).
\newblock {UPPAAL} in a nutshell.
\newblock {\em International Journal on Software Tools for Technology Transfer
  (STTT)}, 1:134--152.

\bibitem[Man et~al., 2011]{ta-radiazioni}
Man, K., Krilavicius, T., Wan, K., Hughes, D., and Lee, K. (2011).
\newblock Modeling and analysis of radiation therapy system with respiratory
  compensation using {UPPAAL}.
\newblock In {\em Proceedings of the 9th IEEE International Symposium on
  Parallel and Distributed Processing with Application - {ISPA} 2011}, pages
  50--54.

\bibitem[{Marques Jr.} et~al., 2013]{ta-ws-bap}
{Marques Jr.}, A.~P., Ravn, A.~P., Srba, J., and Vighio, S. (2013).
\newblock Model-checking web services business activity protocols.
\newblock {\em International Journal on Software Tools for Technology
  Transfer}, 15(2):125--147.

\bibitem[Mendoza et~al., 1999]{boolean-networks-flower}
Mendoza, L., Thieffry, D., and Alvarez-Buylla, E.~R. (1999).
\newblock Genetic control of flower morphogenesis in arabidopsis thaliana: a
  logical analysis.
\newblock {\em Bioinformatics}, 15(7):593--606.

\bibitem[Nakano and Yamaguchi, 2011]{ta-giapponesi}
Nakano, S. and Yamaguchi, S. (2011).
\newblock Two modeling methods for signaling pathways with multiple signals
  using {UPPAAL}.
\newblock In {\em Proceedings of the 2nd International Workshop on Biological
  Processes \& Petri Nets {(BioPPN2011)}}, pages 87--101. CEUR-WS.org.

\bibitem[Reisig, 2011]{petri-nets}
Reisig, W. (2011).
\newblock Petri nets.
\newblock In Koch, I., Reisig, W., Schreiber, F., Dress, A., Vingron, M.,
  Myers, G., Giegerich, R., Fitch, W., and Pevzner, P.~A., editors, {\em
  Modeling in Systems Biology}, volume~16 of {\em Computational Biology}, pages
  37--56. Springer London.
\newblock 10.1007/978-1-84996-474-6\_3.

\bibitem[Russell et~al., 1995]{ras-mekk1}
Russell, M., Lange-Carter, C.~A., and Johnson, G.~L. (1995).
\newblock Direct interaction between {Ras} and the kinase domain of
  mitogen-activated protein kinase kinase kinase ({MEKK1}).
\newblock {\em Journal of Biological Chemistry}, 270(20):11757--11760.

\bibitem[Schivo et~al., 2012]{animo-bibe}
Schivo, S., Scholma, J., Wanders, B., {Urquidi Camacho}, R.~A., van~der Vet,
  P.~E., Karperien, M., Langerak, R., van~de Pol, J., and Post, J.~N. (2012).
\newblock Modelling biological pathway dynamics with {Timed Automata}.
\newblock In {\em 2012 IEEE 12th International Conference on Bioinformatics \&
  Bioengineering (BIBE)}, pages 447--453, Los Alamitos, CA, USA. IEEE Computer
  Society.

\bibitem[Shmulevich and Zhang, 2002]{boolean-networks2}
Shmulevich, I. and Zhang, W. (2002).
\newblock {Binary analysis and optimization-based normalization of gene
  expression data}.
\newblock {\em Bioinformatics}, 18(4):555--565.

\bibitem[Siebert and Bockmayr, 2008]{ta-siebert}
Siebert, H. and Bockmayr, A. (2008).
\newblock Temporal constraints in the logical analysis of regulatory networks.
\newblock {\em Theor. Comput. Sci.}, 391(3):258--275.

\bibitem[Thomas, 1973]{thomas-formalism}
Thomas, R. (1973).
\newblock Boolean formalization of genetic control circuits.
\newblock {\em J. Theor. Biol.}, 42(3):563--585.

\end{thebibliography}





% \end{document}

\input{additional-materials.tex}

\end{document}
